docs/JK.Common/Geospatial/CoordinateBase.md

```
@@ -0,0 +1,244 @@
+[Docs](../../README.md) > [JK.Common](../README.md) > CoordinateBase
+
+# CoordinateBase
+
+**Namespace:** `JK.Common.Geospatial`
+
+Base object for latitudes and longitudes coordinates.
+
+### CoordinateBase
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)